### PR TITLE
Fix some typos to documentation from PR #3134

### PR DESCRIPTION
**Pull request by bartlettroscoe**
@trilinos/framework

This contains a TriBITS snapshot update and another Trilinos commit to fix some documentation typos found in commits in PR #3134.

### CODE REVIEW

File: cmake/tribits/ci_support/filter-packages-list.py [hunk 1]
```diff
@@ -43,17 +43,20 @@
...
   --input-packages-list=<P1>,<P2>,... --keep-test-test-categories=<T1>,<T2>,...
 
-This script takes in a comma-seprated list of TriBITS package names
-<P1>,<P2>,... and then filters out the package names for packages that don't
-the package test test categories listed in <T1>,<T2>,... which are given the
-TriBITS-generated project dependencies file.
+This script takes in a comma-separated list of TriBITS package names in
+--input-packages-list='<P1>,<P2>,...' and then filters out the names for
+packages that don't match the set categories listed in
+--keep-test-test-categories='<T1>,<T2>,...' (where each package's test test
```
Comment (mhoemmen) [inline cmake/tribits/ci_support/filter-packages-list.py]: "test test" (that's OK though, fix later :-) )

File: cmake/tribits/ci_support/filter-packages-list.py [hunk 2]
```diff
@@ -68,8 +71,6 @@
   filter-packages-list.py --keep-test-test-categories=PT,ST,EX [other args]
 
 (or don't both running the script).
```
Comment (mhoemmen) [inline cmake/tribits/ci_support/filter-packages-list.py]: "don't bother"?

File: cmake/tribits/doc/developers_guide/TribitsDevelopersGuide.rst [hunk 3]
```diff
@@ -4505,7 +4512,7 @@ components.  However, detailed descriptions of these processes are deferred to
 the later sections `Pre-push Testing using checkin-test.py`_ and `TriBITS
 CTest/CDash Driver`_.
 
-The standard TriBITS-defined project testing processes are:
+The standard TriBITS-suupported project testing processes are:
```
Comment (mhoemmen) [inline cmake/tribits/doc/developers_guide/TribitsDevelopersGuide.rst]: "supported" (extra "u" there)

File: cmake/tribits/doc/developers_guide/TribitsDevelopersGuide.rst [hunk 4]
```diff
@@ -5051,9 +5057,9 @@ TriBITS packages, one must perform the following actions**:
 
 1) Change the TriBITS CMake files as described above that will result in the
    desired email addresses in the ``CDashSubprojectDependencies.xml``
-   file. One can debug this by running the `checkin-test.py`_ script and
-   seeing what gets written in the generated
-   `<Project>PackageDependencies.xml`_ file in the ``CHECKIN`` directory.
+   file. One can debug this by generating the file
+   `<Project>PackageDependencies.xml`_ as using the cmake -P script
```
Comment (mhoemmen) [inline cmake/tribits/doc/developers_guide/TribitsDevelopersGuide.rst]: "as using" -> "by using"?

File: cmake/tribits/ci_support/filter-packages-list.py (hunk 1 above)
Comment (bartlettroscoe) [reply cmake/tribits/ci_support/filter-packages-list.py]: @mhoemmen, "test test categories" is actually not a typo.  See:

* https://tribits.org/doc/TribitsDevelopersGuide.html#test-test-category

**Comment by bartlettroscoe** [comment]
@mhoemmen, thanks for the review!  I have fixed these typos is the TriBITS commit TriBITSPub/TriBITS@0681524fc58529e940ff581e1a6ee086b62d02b9 and it is already reflected in the auto updated TriBITS documentation at:

* https://tribits.org/doc/TribitsDevelopersGuide.html

The next time TriBITS 'master' get snapshotted to Trilinos, Trilinos's copy of these will get these fixes too.

Thanks!

**Pull request merged**